cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CompileTask.java

```diff
@@ -67,24 +71,52 @@ public void execute(Project project) {
         System.setProperty(CentralClientConstants.ENABLE_OUTPUT_STREAM, "true");
 
         try {
+            List<Diagnostic> diagnostics = new ArrayList<>();
             long start = 0;
             if (project.buildOptions().dumpBuildTime()) {
                 start = System.currentTimeMillis();
-                project.currentPackage().getResolution();
+                BuildTime.getInstance().packageResolutionDuration = System.currentTimeMillis() - start;
```

Comment:
I can see `BuildTime.getInstance().packageResolutionDuration = System.currentTimeMillis() - start;` twice here.  Could you please explain the logic?

Reply:
It's a mistake. Refactored with [5d39000](https://github.com/ballerina-platform/ballerina-lang/pull/35411/commits/5d390009c6c3d3ab3753b5be2102537d6c6d53bb)